python/Ganga/Lib/Localhost/LocalHostExec.py

```diff
@@ -88,14 +88,18 @@
 
 import time #datetime #disabled for python2.2 compatiblity
 
+if os.path.isfile(os.path.abspath(appscriptpath[0])):
```

Comment:
And now this drops the ability for me to have a string and call a DiracFile... I'll just add an issue for that.